web/src/stream_popover.ts

```diff
@@ -345,6 +345,7 @@ export async function build_move_topic_to_stream_popover(
     message?: Message,
```

Comment:
"Refactor" is not a correct word to use in this commit message; it is only correct if there are no functional changes.

"restructure" would be a fine word to use.

Reply:
Sure. Thank you for the feedback. I will surely remember this in the future.